### PR TITLE
Add CDN source

### PR DESCRIPTION
**Pull request by VladimirSlavik**
Not sure what to have in tests...

### CODE REVIEW

File: pyanaconda/modules/payloads/source/cdn/cdn.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+from pyanaconda.core.i18n import _
+from pyanaconda.modules.payloads.constants import SourceType
+from pyanaconda.modules.payloads.source.repo_files.repo_files import RepoFilesSourceModule
+from pyanaconda.modules.payloads.source.cdn.cdn_interface import CdnSourceInterface
+from pyanaconda.anaconda_loggers import get_module_logger
+
+log = get_module_logger(__name__)
+
+
+class CdnSourceModule(RepoFilesSourceModule):
```
Comment (jkonecny12) [inline pyanaconda/modules/payloads/source/cdn/cdn.py:29]: I think it should be CDN with upper case all the time.

File: pyanaconda/modules/payloads/source/cdn/cdn_interface.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#
+# DBus interface for payload Repo files image source.
```
Comment (jkonecny12) [inline pyanaconda/modules/payloads/source/cdn/cdn_interface.py:2]: Please remove this or change the content.

File: pyanaconda/modules/payloads/source/cdn/cdn.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
+#
+# Kickstart module for Repo files payload source.
```
Comment (jkonecny12) [inline pyanaconda/modules/payloads/source/cdn/cdn.py:2]: The same here.

**Comment by VladimirSlavik** [comment]
Update: resolved the uppercase name and leftover comments.

File: pyanaconda/modules/payloads/source/cdn/cdn.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    def description(self):
+        """Get description of this source."""
+        return _("Red Hat CDN")
+
+    def __repr__(self):
+        return "Source(type='CDN')"
+
+    def for_publication(self):
+        """Get the interface used to publish this source."""
+        return CDNSourceInterface(self)
```
Comment (jkonecny12) [inline pyanaconda/modules/payloads/source/cdn/cdn.py:47]: Please move this methods to the top of the class based on https://github.com/rhinstaller/anaconda/pull/2514

Comment (VladimirSlavik) [reply pyanaconda/modules/payloads/source/cdn/cdn.py]: Done!

**Pull request merged**